press/press/doctype/team/team.py

```diff
@@ -240,10 +240,14 @@ def on_update(self):
 		self.validate_payment_mode()
 		self.update_draft_invoice_payment_mode()
 
-		if not self.is_new() and self.billing_name and not frappe.conf.allow_tests:
+		if not self.is_new() and self.billing_name \
+			and not frappe.conf.allow_tests and not self.is_team_owned_by_admin():
```

Comment:
this conf key should never be used in code? :thinking: It just means if frappe can run tests or not.